packages/wrangler/src/images/local.ts

```diff
@@ -0,0 +1,187 @@
+import { File } from "buffer";
+import sharp from "sharp";
+import { z } from "zod";
+import type { ImageInfoResponse } from "@cloudflare/workers-types/experimental";
+import type { Sharp } from "sharp";
+
+const Transform = z.object({
+	imageIndex: z.number().optional(),
+	rotate: z.number().optional(),
+	width: z.number().optional(),
+	height: z.number().optional(),
+});
+
+const Transforms = z.array(Transform);
```

Comment:
Adding Zod is a fairly significant bloating of Wrangler's bundle size for what seems like just these two validations. Would it be infeasible to do these manually?